base/idset.jl

```diff
@@ -5,14 +5,13 @@
     IdSet()
 
 IdSet{T}() constructs a set (see [`Set`](@ref)) using
-`===` as equality with values of type `V`.
+`===` as equality with values of type `T`.
 
-In the example below, the values are all `isequal` so they get overwritten.
-The `IdSet` compares by `===` so preserves the 3 different keys.
-
-Examples
-≡≡≡≡≡≡≡≡
+In the example below, the values are all `isequal` so they get overwritten in the ordinary `Set`.
+The `IdSet` compares by `===` and so preserves the 3 different values.
 
+# Examples
+```jldoctest; filter = r"^\\s*([1\\.0]+|true)"
```

Comment:
One minor change I'd reccomend:

```suggestion
```jldoctest; filter = r"^  (1|1\.0|true)"
```
Shorter, stricter, and more readable.

Reply:
But then it's sensitive to the amount of indentation printed in the set which I think shouldn't matter